### ISSUE TITLE
AddConditionalFormat() combines formatting into one

### ISSUE THREAD

**Issue by amelkor** [ClosedXML/ClosedXML#973]
- [x] Bug
- [ ] Feature

**Version of ClosedXML**

0.93.1

**What is the current behavior?**

AddConditionalFormat() when used for different ranges combines ranges together

**What is the expected behavior or new feature?**

AddConditionalFormat() adds conditional formatting only for specified cells range

**Did this work in previous versions of our tool?  Which versions?**

No

## Reproducibility
**Code to reproduce problem:**
```c#
/*Expected: set conditional format only for ranges:
* A1:E1
* A4:E4
* A7:E7
* A10:E10
* What we will really get: conditional format set for range A1:E10
*/
using System.Linq;
using ClosedXML.Excel;

class Program
{
    static void Main(string[] args)
    {
        var templatePath = @"C:\Temp\test1.xltx";
        var savePath = @"C:\Temp\test_result.xlsx";

        using (var book = XLWorkbook.OpenFromTemplate(templatePath))
        {
            string rangesStr = "A1:E1,A4:E4,A7:E7,A10:E10";


            var sheet = book.Worksheets.First();
            foreach (var range in sheet.Ranges(rangesStr))
            {
                range.AddConditionalFormat().WhenNotBlank()
                    .Fill.SetBackgroundColor(XLColor.Red)
                    .Font.SetFontColor(XLColor.White)
                    .Font.SetBold(true);
            }
            book.SaveAs(savePath);
        }
    }
}
```

Test excel template file: [test1.zip](https://github.com/ClosedXML/ClosedXML/files/2282860/test1.zip)

Template looks like:
![exceltemplatepic](https://user-images.githubusercontent.com/30626539/44031186-d5de84be-9f0b-11e8-8b06-34183959ab8e.JPG)

Result we got:
![excelresultpic](https://user-images.githubusercontent.com/30626539/44031192-da0a4b18-9f0b-11e8-96ca-55afa81349e4.JPG)

Expected result:
![excelexpectedpic](https://user-images.githubusercontent.com/30626539/44031194-dca2a244-9f0b-11e8-8b6c-357022b999fc.JPG)

**Comment by mrUlrik** [comment]
I am currently using 0.94.2 and this problem still exists.

Regardless of how I use AddConditionalFormat, through a loop or manually specify the exact ranges, separately, the end result is the range is expanded across the minimum and maximum values across the enter table.

More specifically, I'd like a 3-color scale for each column in a table. Each column has it's own maximum and minimum value to grade the color used in each cell. When attempting this, it will take a single maximum and minimum value from the combined data of every column with a separate conditional format applied.

When viewing the rules in Excel, it is clearly reporting in a single rule, all of the columns in my table.

**Comment by Pankraty** [comment]
Such kind of condition formatting rules was excluded from a consolidation in #1203 and will be released in 0.95. Try latest beta, it must solve your issue.

**Comment by mrUlrik** [comment]
Thank you for the quick return! I'm now using 0.95.0-beta1. Unfortunately it's not quite there yet.

I have times in the middle of the table, the rest is percentages. This produces two Conditional Formatting Rules for five columns:
=$A$1:$C$13,$F$1:$F$13
=$D$1:$E$13

[Sheet.xlsx](https://github.com/ClosedXML/ClosedXML/files/3486246/Sheet.xlsx)
![image](https://user-images.githubusercontent.com/24566356/62777793-831df380-ba7c-11e9-91a5-f0f7d48297a5.png)
